.github/workflows/build-and-deploy-container-image.yaml

```
@@ -221,8 +221,21 @@ on:
       github-token:
         description: Token used for accessing GitHub.com
         required: true
+      asana-github-api-token:
+        description: Token used for asana github api access
+        required: false
 
 jobs:
+  
+  update-asana: 
+    name: Update asana tasks
+    runs-on: ubuntu-latest
+    if: ${{ secrets.asana-github-api-token }} != ''
```

Ich glaub hier reicht
```suggestion
    if: ${{ secrets.asana-github-api-token }}
```